pkg/labelsfilter/filter_test.go

```diff
@@ -162,3 +164,36 @@ func (s *LabelsPrefCfgSuite) TestFilterLabelsDocExample(c *C) {
 	c.Assert(len(filtered), Equals, 6)
 	c.Assert(filtered, checker.DeepEquals, wanted)
 }
+
+func TestFilterLabelsByRegex(t *testing.T) {
+	type args struct {
+		excludePatterns []*regexp.Regexp
+		labels          map[string]string
+	}
+	tests := []struct {
```

Comment:
We could use a map here and leverage the use case name as the key.
Additionally we are only testing the happy path.

Reply:
Good catch. Added a new case for testing the baseline.
The array pattern seems to be a standard everywhere else though ? My tests follow the same pattern.